swarm/storage/dbstore_test.go

```diff
@@ -189,3 +192,74 @@ func TestDbStoreSyncIterator(t *testing.T) {
 		t.Fatalf("Expected %v chunk, got %v", keys[3], res[0])
 	}
 }
+
+func initMockDbStore(t *testing.T, mockStore *mock.NodeStore) *DbStore {
+	dir, err := ioutil.TempDir("", "bzz-storage-test-mock")
+	if err != nil {
+		t.Fatal(err)
+	}
+	m, err := NewMockDbStore(dir, MakeHashFunc(SHA3Hash), defaultDbCapacity, defaultRadius, mockStore)
+	if err != nil {
+		t.Fatal("can't create store:", err)
+	}
+	return m
+}
+
+func testMockDbStore(l int64, branches int64, t *testing.T) {
```

Comment:
I would welcome a quick description of what the test does (even if it seems obvious :) )

Reply:
Yes, I thought that it is obvious, so I did not put a comment. I will.